cli/lib/tasks/info.js

```diff
@@ -16,44 +18,77 @@ const getPlatformExecutable = () => {
   }
 }
 
-const getInstallationDir = () => {
+const getInstallationDir = (version = util.pkgVersion()) => {
+  return getInfoFileContents()
+  .tap(debug)
+  .get('install_directory')
+  .catch(() => {
+    let cache_directory = cachedir('Cypress')
```

Comment:
should we use capital letter? I would prefer lower case for simplicity

Reply:
I believe its canonical to capitalize the name. Looking through my `~/Library/Application Support` and `/Library/Caches` most apps (including Electron) have a capitalized folder containing the cached + app data.

Reply:
![screen shot 2018-04-14 at 3 00 39 pm](https://user-images.githubusercontent.com/1268976/38771665-ee2eb9f6-3ff4-11e8-8ca4-2b2cae2a57cb.png)